src/main/java/core/basesyntax/service/file/work/CsvFileWriterServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.file.work;
+
+import core.basesyntax.model.FruitTransaction;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+public class CsvFileWriterServiceImpl implements CsvFileWriterService {
+    @Override
+    public void writeToFile(String writeToFilePath, List<FruitTransaction> data) {
+        try (PrintWriter printWriter = new PrintWriter(new FileWriter(writeToFilePath))) {
+            data.stream()
+                    .map(this::convertToCsv)
+                    .forEach(printWriter::println);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to this path: "
+                    + writeToFilePath, e);
+        }
+    }
+
+    private String convertToCsv(FruitTransaction transaction) {
```

Comment:
This logic should be in FruitTransactionReportMakerImpl. CsvFileWriterServiceImpl should only write to file.